### PR TITLE
fix(Tab): can't scroll when active item is inside view

### LINKED ISSUE #226: [Tab]Tab won't slide when active one is visible
- [x] I have searched the [issues](https://github.com/alibaba-fusion/next/issues) of this repository and believe that this is not a duplicate.

### Version
1.11.11

### Component
Tab

### Environment
Mac OS X

### Reproduction link
[https://codepen.io/guanpu/pen/xMGYKB](https://codepen.io/guanpu/pen/xMGYKB)

### Steps to reproduce
In above example, just click the "prev" button, the toLeft icon, and it won't work

### PR DESCRIPTION
**Pull request by guanpu**
Fixes #226 

The basic idea is only auto scroll to active tab when the item mount, not auto scroll when the component re-renders

### CODE REVIEW

File: src/tab/tabs/nav.jsx [hunk 1]
```diff
@@ -45,21 +45,22 @@ class Nav extends React.Component {
...
 
+        ctx.setSlideBtn();
+        ctx.getDropdownItems(this.props);
         // 此处通过延时处理，屏蔽动画带来的定位不准确问题（由于要支持ie9，因此无法使用transitionend）
         clearTimeout(ctx.scrollTimer);
         ctx.scrollTimer = setTimeout(() => {
             ctx.scrollToActiveTab();
         }, 400);
 
+        events.on(window, 'resize', this.onWindowResized);
```
[the remark this answers is not in the document]
Comment (guanpu) [reply src/tab/tabs/nav.jsx]: That makes sense, I'll add it.

[the remark this answers is not in the document]
Comment (guanpu) [reply src/tab/tabs/nav.jsx]: Ah, `this.onWindowResized` is already a throttled function if you look at https://github.com/alibaba-fusion/next/blob/4631523db6d60949ba6063a2ff0c36807fa880d3/src/tab/tabs/nav.jsx#L313

**Comment by guanpu** [comment]
Please don't merge this, I find another related bug and needs to be fixed at this MR.

**Comment by guanpu** [comment]
I would like to fix #405 in this MR, so please wait for next commit.

**Comment by guanpu** [comment]
@no-repeat Please review the changes, the basic idea is to make dropdown to contain **ALL** items(if excess) or none(if tabs item's width not excess).

**Comment by youluna** [comment]
@guanpu  merge master and resolve conflicts please~

**Pull request merged**